### ISSUE TITLE
EDSC-874: Going back to collections list from other pages in IE won't trigger URL reload

### ISSUE THREAD

**Issue by earthdata-tasktop** [nasa/edsc-search-test-config-3#38]

**Comment by earthdata-tasktop** [comment]
(Comment from Kathleen Carr):

[~bo.sun][~pquinn]I am trying to verify this ticket.   There is no description of the problem in the ticket, so I'm not sure what the behavior is that is incorrect.  

I tried the scenario that Bo suggested in the Verify comment in both OPS and UAT, but I am not seeing any different behavior between the two environments.  

Could you please give me more information regarding what the bad behavior is in OPS that we have fixed in UAT?

**Comment by earthdata-tasktop** [comment]
(Comment from Bo Sun):

The incorrect behavior is that if you are on a page other than collection list (i.e. granule list or project page or collection details, etc.) and do a text search, you should be are not sent back to the collection page. The URL and the content of the collection list should also change.

**Comment by earthdata-tasktop** [comment]
(Comment from Bo Sun):

##### Verify

In IE

* from granule list page, type in a search keyword in the text box. e.g. "MODIS"
* EDSC should send you back to collection list and *perform the search*
* from project list page, type in a search keyword
* EDSC should also send you back to collection list and perform the search

**Comment by earthdata-tasktop** [comment]
(Comment from Kathleen Carr):

[~bo.sun][~pquinn]I am trying to verify this ticket.   There is no description of the problem in the ticket, so I'm not sure what the behavior is that is incorrect.  

I tried the scenario that Bo suggested in the Verify comment in both OPS and UAT, but I am not seeing any different behavior between the two environments.  

Could you please give me more information regarding what the bad behavior is in OPS that we have fixed in UAT?

**Comment by earthdata-tasktop** [comment]
(Comment from Bo Sun):

##### Verify

In IE

* from granule list page, type in a search keyword in the text box. e.g. "MODIS"
* EDSC should send you back to collection list and *perform the search*
* from project list page, type in a search keyword
* EDSC should also send you back to collection list and perform the search

**Comment by earthdata-tasktop** [comment]
(Comment from Dana Shum):

Migrating "Internal Verified" tickets to "Completed".

**Comment by earthdata-tasktop** [comment]
(Comment from Kathleen Carr):

I tried the two scenarios listed in the comment with the Verify instructions (i.e., from granule list page and from project list page).  

In both cases, EDSC sent me back to the collection list, and the url changed from search/granules to  search/collections, and from search/projects to search/collections respectively.

I performed my verification in UAT, but I also noticed that the fix has already been deployed to OPS.

**Comment by earthdata-tasktop** [comment]
(Comment from Dana Shum):

Migrating "Internal Verified" tickets to "Completed".

**Comment by earthdata-tasktop** [comment]
(Comment from Kathleen Carr):

I tried the two scenarios listed in the comment with the Verify instructions (i.e., from granule list page and from project list page).  

In both cases, EDSC sent me back to the collection list, and the url changed from search/granules to  search/collections, and from search/projects to search/collections respectively.

I performed my verification in UAT, but I also noticed that the fix has already been deployed to OPS.

**Comment by earthdata-tasktop** [comment]
(Comment from Kathleen Carr):

I tried the two scenarios listed in the comment with the Verify instructions (i.e., from granule list page and from project list page).  

In both cases, EDSC sent me back to the collection list, and the url changed from search/granules to  search/collections, and from search/projects to search/collections respectively.

I performed my verification in UAT, but I also noticed that the fix has already been deployed to OPS.
